hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java

```diff
@@ -1202,6 +1287,10 @@ public String getCanonicalServiceName() {
     return this.statistics;
   }
 
+  public void setListenerOperation(String operation) {
```

Comment:
This will only be used by test code ? If so, lower accessibility to private and set VisibleForTesting.

Reply:
done